pkg/webui/console/containers/device-title-section/device-title-section.js

```diff
@@ -36,7 +37,7 @@ const m = defineMessages({
   uplinkDownlinkTooltip:
     'The number of sent uplinks and received downlinks of this end device since the last frame counter reset.',
   lastSeenAvailableTooltip:
-    'The elapsed time since the network registered the last activity of this end device. This is determined from sent uplinks, confirmed downlinks or (re)join requests.',
+    'The elapsed time since the network registered the last activity of this end device. This is determined from sent uplinks, confirmed downlinks or (re)join requests. The last activity was received at {lastActivityInfo}',
```

Comment:
```suggestion
    'The elapsed time since the network registered the last activity of this end device. This is determined from sent uplinks, confirmed downlinks or (re)join requests.{break}The last activity was received at {lastActivityInfo}',
```

Please insert a line break.